pkg/didcomm/transport/ws/pool.go

```diff
@@ -18,6 +19,11 @@ import (
 	"github.com/hyperledger/aries-framework-go/pkg/didcomm/transport"
 )
 
+const (
+	// TODO configure ping request frequency
+	pingFrequency = 20 * time.Second
```

Comment:
Longer frequency didn’t work?

Reply:
We should be able to configure this frequency (TODO).